src/Value/Value.php

```diff
@@ -70,20 +70,27 @@ public static function parseValue(ParserState $oParserState, array $aListDelimit
             if (count($aStack) === 1) {
                 return $aStack[0];
             }
-            $iStartPosition = null;
-            while (($iStartPosition = array_search($sDelimiter, $aStack, true)) !== false) {
+            $aNewStack = [];
+            $aStackLength = count($aStack);
```

Comment:
`$aStackLength` can be initialized before and used in the preceding early-return test.

Reply:
And maybe should be named `$iStackLength`.

Reply:
Yep, good catch. I had forgotten the variable naming scheme.

Reply:
That variable naming scheme may be changed in future - https://github.com/MyIntervals/PHP-CSS-Parser/issues/450#issuecomment-1931576970 - though until such time it's best to maintain consistency.